classes/03class/exercises/c03-docker06/feernandobraga/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM alpine:latest
```

Comment:
can you not use latest? and also provide some explanation on why you should avoid using it?

Reply:
Hi Caio,
Yes, I could specify a version instead of using latest.
This would prevent my image from breaking in case a new version is released and it doesn't support my legacy code/application. It would also prevent that the entire image is downloaded over and over again every time a new version is released.
However, from the security perspective, by using the latest tag I'm certain that I will always receive patches and fixes for recent vulnerabilities. Isn't it worth focusing on security?